task3/src/Classes/Models/Fibonacci.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Models;
+
+class Fibonacci
+{
```

Comment:
Public methods first, then protected, then private.
phpDocs on class and methods.
Why do you need class, if you don't utilize properties, constructor and `this`?